[CIAPP-5225] Add information about flaky facets in Test Runs page. (#15866)
DataDog/documentation: content/en/continuous_integration/guides/flaky_test_management.md

## Watch for new flaky tests

These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.

### Test Runs page

1. Navigate to the [Test Runs][1] page.
2. In the facets list on the left sidebar, expand the **New Flaky** facet in the **Test** section, and check `true`.

> All test runs that exhibited flakiness behavior for the first time as per the definition above are displayed.

Reviewer: ```suggestion
All test runs that exhibited flakey behavior for the first time as per the definition above are displayed.
```

## Watch for new flaky tests

These are tests that exhibit flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository.

> ### In the Test Runs page

Reviewer: ```suggestion
### Test Runs page
```

> 1. In the facets list on the left sidebar, in the "Test" section, expand the "New Flaky" facet and check `true`

Reviewer: ```suggestion
1. Navigate to the [Test Runs][1] page.
2. In the facets list on the left sidebar, in the **Test** section, expand the **New Flaky** facet and check `true`.
```

> 2. This displays all test runs that exhibited flakiness behavior for the first time as per the definition above.

Reviewer: ```suggestion
All test runs that exhibited flakiness behavior for the first time as per the definition above are displayed.
```

> ### In the Branches page

Reviewer: ```suggestion
### Branches page
```

## Watch for known flaky failed tests

Known flaky failed tests are tests that have flaky behavior on the current or default branch of the repository.

### In the Test Runs page

1. In the facets list on the left sidebar, in the "Test" section, expand the "Known Flaky" facet and check `true`
2. Failed test runs that were known to be flaky as per the definition above are displayed.

### In the Branches page

1. On the Tests page, select the **Branches** view.

2. Filter the table to see any branches, services, or commits of interest.

3. The **Failed** column contains the number of failed tests and known flaky failed tests in the latest commit.

> {{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}

[1]: https://app.datadoghq.com/ci/test-runs
[2]: https://app.datadoghq.com/ci/test-services
```

Author: should the app URL differ depending on the chosen datacenter?
